packages/passwordless-auth-construct/src/sign-up/create_user_service.ts

```diff
@@ -0,0 +1,103 @@
+// ../amplify-backend/packages/passwordless-auth-construct/lib/sign-up/createUserService.js
+
+import {
+  AdminCreateUserCommand,
+  CognitoIdentityProviderClient,
+} from '@aws-sdk/client-cognito-identity-provider';
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
+import { PASSWORDLESS_SIGN_UP_ATTR_NAME } from '../constants.js';
+
+/**
+ * The Create User for Passwordless SignUp lambda handler
+ * @param event - The event provided by ApiGateway for the request
+ * @returns the response, including status code and response body
+ */
+export const createUser = async (
+  event: APIGatewayProxyEvent
+): Promise<APIGatewayProxyResult> => {
+  try {
+    const params = event && event.body && JSON.parse(event.body);
+
+    if (
+      !params ||
+      (!params.email && !params.phone_number) ||
+      !params.userPoolId ||
+      !params.region ||
+      !params.username
+    ) {
+      return {
+        statusCode: 400,
+        body: 'Missing parameters',
+      };
+    }
+
+    // this is the format for Cognito UserPool API, eslint doesnt like it otherwise
+    // eslint-disable-next-line
+    const userAttributes: Array<{ Name: string; Value: string }> = [];
+    const passwordlessConfiguration: {
+      allowSignInAttempt: boolean;
+      deliveryMedium?: 'SMS' | 'EMAIL';
```

Comment:
```suggestion
      deliveryMedium?: DeliveryMedium;
```
request: use the common enum